FluentFlyoutWPF/App.xaml

```diff
@@ -9,12 +9,11 @@
     <Application.Resources>
         <ResourceDictionary>
             <ResourceDictionary.MergedDictionaries>
-                <ui:ThemesDictionary Theme="Light" />
```

Comment:
Adding `<ui:ThemesDictionary Theme="Unknown"/>` will fix some issues related to theming